examples/category.tf

```diff
@@ -0,0 +1,8 @@
+provider "logdna" {
```

Comment:
Don't for get documentation in the `docs` directory too.  Just examples isn't enough.  At a minimum, there should be something in `docs/resources`